docs/language.md

```diff
@@ -6444,26 +6657,46 @@ Although optional, each block serves a specific purpose when executing a transac
 and it is recommended that developers use these blocks when creating their transactions.
 The following will detail the purpose of and how to use each block.
 
-## Prepare
+### Transaction Parameters
+
+Transactions may declare parameters.
+Transaction parameters are declared like function parameters.
+The arguments for the transaction are passed in the sent transaction.
 
-The `prepare` **block** is used when access to the private `AuthAccount` object of **signing accounts** is required for your transaction.
+Transaction parameters are accessible in all blocks.
```

Comment:
Sometimes we use the term phase, and sometimes we use the term block when describing the parts of a transaction. I would prefer the term phase, but we at least should be consistent. What do you think, @turbolent ?
```suggestion
Transaction parameters are accessible in all the phases of a transaction.
```

Reply:
Agreed. It was rewritten to use blocks in most places, but yeah, let's use phase everywhere

Reply:
I checked the document again, and it is actually using block elusively and there is a single explanation using the term phase:

> Then, three optional main phases: Preparation, execution, and postconditions, only in that order.
> Each phase is a block of code that executes sequentially.

So this is already consistent and I'm not sure replacing "block" with "phase" everywhere changes much

Reply:
I just feel like phase is a more accurate term, because it illustrates that they are groupings of actions that happens in an order. Each phase contains a block of code though, but block doesn't indicate an order, so that is why I like phase more. Does that make sense? It isn't that big of a deal though. If you want to keep it as block, I understand

Reply:
Oh, I hadn't thought of it that way, that's a good argument! I'll replace

Reply:
Improved it in 0c7b5f8